### PR TITLE
Feature/36196 fix jsdoc issues on admin/services/data-access

### PR DESCRIPTION
**Pull request by GuyHarwood**
down to 46 issues

### CODE REVIEW

File: admin/services/data-access/sce.data.service.js [hunk 1]
```diff
@@ -37,15 +37,15 @@ sceDataService.sqlFindSceSchools = async () => {
   FROM ${sqlService.adminSchema}.[school]
   LEFT JOIN ${sqlService.adminSchema}.[sce]
     ON sce.school_id = school.id
-  WHERE school.leaCode = 702 
+  WHERE school.leaCode = 702
```
Comment (jon-shipley) [inline admin/services/data-access/sce.data.service.js]: I realise this is existing code - but why is there a magic number in there?

Comment (GuyHarwood) [reply admin/services/data-access/sce.data.service.js]: that's bizarre!

Comment (GuyHarwood) [reply admin/services/data-access/sce.data.service.js]: @GeorgeChatzigiannis any idea on this one?

Comment (GeorgeChatzigiannis) [reply admin/services/data-access/sce.data.service.js]: seems like all the sce schools have a lea code of 702 as per https://github.com/DFEAGILEDEVOPS/MTC/pull/1064/files

Aside from the magic number I don't really like the naming convention used on this service as well.

Comment (GuyHarwood) [reply admin/services/data-access/sce.data.service.js]: ok, ill refactor that out to a variable to make it obvious.  thanks

**Pull request merged**